.github/workflows/CI-python.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           ["responsibleai", "erroranalysis", "raiutils", "rai_test_utils"]
         operatingSystem:
           [ubuntu-20.04, ubuntu-latest, macos-latest, windows-latest]
-        pythonVersion: ["3.6", "3.7", "3.8", "3.9", "3.10"]
```

Comment:
The change to remove 3.6 could be committed as a separate PR. Might be good for tracking purpose.

Reply:
Sure - https://github.com/microsoft/responsible-ai-toolbox/pull/2249